route/build/src/write_restart_pio.f90

```diff
@@ -23,16 +25,19 @@ MODULE write_restart_pio
 USE public_var,        ONLY: iulog             ! i/o logical unit number
 USE public_var,        ONLY: integerMissing
 USE public_var,        ONLY: realMissing
+USE public_var,        ONLY: tracer             ! tracer logical
 ! meta data
 USE globalData,        ONLY: meta_stateDims  ! states dimension meta
 USE globalData,        ONLY: meta_qDims
 USE globalData,        ONLY: meta_irf_bas
+USE globalData,        ONLY: meta_bas_tracer
```

Comment:
What does this variable mean? It maybe needs a comment...

Reply:
`meta_bas_tracer` is changed to `meta_bas_solute` and added comments (for all the meta data)